pkg/server/middleware/middleware.go

```diff
@@ -87,7 +87,7 @@ func WithProviderToken(jwtClient auth.JWTClient, h http.Handler, log logr.Logger
 		tokenSlice := strings.Split(tokenStr, "token ")
 
 		if len(tokenSlice) < 2 {
-			log.V(logger.LogLevelDebug).Info("missing or invalid token.")
+			log.V(logger.LogLevelDebug).Info("missing or invalid git-provider-token.")
```

Comment:
Could we improve this with the guidance above?

https://github.com/weaveworks/weave-gitops/pull/3308/files#diff-1ea5dccd5193cc193367354f068acde964eb10df67d3201f7cd63b0757aaecf5R73